CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -63,6 +63,30 @@ async def manual_report(state_file: UploadFile = File(None), cdc_file:  UploadFi
     return res
 
 
+@app.get("/reports/{report_id}")
+async def get_report(report_id: int):
+    """
+    Endpoint to fetch a report by its ID.
+    """
```

Comment:
This can be condensed into 1 line like so: #Endpoint to fetch a report by its ID.